python/arcticc/__init__.py

```diff
@@ -1 +1,9 @@
+# file from ArcticDB
```

Comment:
This line is redundant

Reply:
I've put a similar line in all the namespace package `__init__.py` files to aid debugging by identifying the source project.

Reply:
No, but you can still inspect the installed file:

```
import arcticc
arcticc.__file__
```